website/client/src/components/hall/index.vue

```diff
@@ -1,34 +1,37 @@
 <template>
   <div class="row">
-    <secondary-menu class="col-12">
-      <router-link
-        class="nav-link"
-        :to="{name: 'contributors'}"
-        exact="exact"
-        :class="{'active': $route.name === 'contributors'}"
-      >
-        {{ $t('hallContributors') }}
-      </router-link>
-      <router-link
-        class="nav-link"
-        :to="{name: 'patrons'}"
-        :class="{'active': $route.name === 'patrons'}"
-      >
-        {{ $t('hallPatrons') }}
-      </router-link>
-    </secondary-menu>
+    <navbar
+      class="secondary-menu col-12"
+      :items="tabs"
+      router-links
+    />
     <div class="col-12">
       <router-view />
     </div>
   </div>
 </template>
 
 <script>
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
+    tabs () {
```

Comment:
data prop

Reply:
i think computed property better for few reasons:
- computed better because you may need to add conditions of show for some tabs;
- you can incapsulate this property in mixin later;
- i don't want to allow somebody to add tab in method - it's bad practice (i think);

Reply:
That's true @Trigur but right now there are no conditions, if we ever add them we can move the prop to a computed later.

> i don't want to allow somebody to add tab in method - it's bad practice (i think);

that's right! methods should be avoided when only used to calculate some data